src/lib/dnssd/Resolver_ImplMinimalMdns.cpp

```diff
@@ -642,6 +665,11 @@ CHIP_ERROR MinMdnsResolver::DiscoverCommissioners(DiscoveryFilter filter)
     return BrowseNodes(DiscoveryType::kCommissionerNode, filter);
 }
 
+CHIP_ERROR MinMdnsResolver::DiscoverOperational(DiscoveryFilter filter)
+{
+    return BrowseNodes(DiscoveryType::kOperational, filter);
```

Comment:
I am pretty sure if a "targeted" resolve now happens in the middle of that browse the browse will be silently canceled, no?

Reply:
Actually I'm not sure, but I should unit test this.

Reply:
Did you test both platform mdns (darwin/bonjour and linux/avahi) and minmdns?